Components/Modal/modal.js

```diff
@@ -0,0 +1,16 @@
+const btnAction = document.querySelector('#btn-actionModal');
+const btnInfo = document.querySelector('#btn-infoModal');
+const actionModal = document.querySelector('#actionModal');
+const closeModalBtn = document.querySelector('#closeModalBtn');
+
+const actionModalHandler = () => {
+    actionModal.classList.add("modalShow");
```

Comment:
You can also explore the use of toggle() property as well, having a click event listener to handle it